requirements.txt

```diff
@@ -0,0 +1,5 @@
+cortx-hare
```

Comment:
Shall we maintain this file as dependencies.txt, to avoid confusing this with python dependencies, if at all in future. This file is treated anyways differently by setup.py

Reply:
This is not python dependency. We need cortx-hare to be installed, because we are going to call hare interface, hctl to fetch the fids. If we are not using it today, then we have choice to add it later or now.

Reply:
Am not saying these are python dependencies. so using file naming convention used for python requirements usually, for non-python dependencies can confuse later

Reply:
no functional issue, am fine